static/coffee/controllers.coffee

```diff
@@ -827,29 +834,71 @@ controllers.controller('DateRangeController', ['$scope', ($scope) ->
 #============================================================================
 # Pod controller
 #============================================================================
-controllers.controller('PodController', ['$scope', 'PodApi', ($scope, PodApi) ->
+controllers.controller('PodController', ['$q', '$scope', 'PodApi', ($q, $scope, PodApi) ->
   $scope.init = (podId, caseId, podConfig) ->
     $scope.podId = podId
     $scope.caseId = caseId
     $scope.podConfig = podConfig
+
+    # TODO handle api failures
     $scope.update()
```

Comment:
The plan is to handle a failed initial pod load differently. The plan is to handle this when we change the UI to show when pods are still loading.